src/contract-deployment/config.ts

```diff
@@ -110,6 +110,10 @@ export const makeContractDeployConfig = async (
         )
       },
     },
+    OVM_MultiRelay: {
+      factory: getContractFactory('OVM_MultiRelay'),
+      params: [await AddressManager.getAddress('Proxy__OVM_L1CrossDomainMessenger')]
```

Comment:
This results in a race condition when not waiting for the transactions and could initialize the relay to the zero address